src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -25,23 +25,25 @@
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"), new Gender("M"),
-                getTagSet("friends"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456B")),
+                getStudentGroupSet("friends"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456B")),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"), new Gender("F"),
-                getTagSet("colleagues", "friends"), new Block("A"),
+                getStudentGroupSet("colleagues", "friends"), new Block("A"),
                     new Room("405"), new MatriculationNumber("A0123456C")),
             new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
                 new Address("Blk 11 Ang Mo Kio Street 74, #11-04"), new Gender("F"),
-                getTagSet("neighbours"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456D")),
+                getStudentGroupSet("neighbours"), new Block("A"), new Room("405"),
+                    new MatriculationNumber("A0123456D")),
             new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
                 new Address("Blk 436 Serangoon Gardens Street 26, #16-43"), new Gender("M"),
-                getTagSet("family"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456E")),
+                getStudentGroupSet("family"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456E")),
             new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
                 new Address("Blk 47 Tampines Street 20, #17-35"), new Gender("M"),
-                getTagSet("classmates"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456F")),
+                getStudentGroupSet("classmates"), new Block("A"), new Room("405"),
+                    new MatriculationNumber("A0123456F")),
             new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
                 new Address("Blk 45 Aljunied Street 85, #11-31"), new Gender("M"),
-                getTagSet("colleagues"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456G"))
+                getStudentGroupSet("colleagues"), new Block("A"), new Room("405"), new MatriculationNumber("A0123456G"))
```

Comment:
maybe can change these to example student groups like basketball, soccer, choir etc

Reply:
Good point, will update it.